src/main/java/seedu/address/model/ModelManager.java

```diff
@@ -177,5 +180,25 @@ public boolean hasMeetingOverlap(Meeting meeting) {
                 .anyMatch(existingMeeting -> existingMeeting.overlapsWith(meeting));
     }
 
-
+    public void removeExpiredMeetings() {
```

Comment:
Loops through the person lists and loops through the meeting list for each person to filter for meetings that are not expired before setting each person with those meetings.